spec/dummy-app/spec/lib/devise_recognisable/guard_spec.rb

```diff
@@ -37,10 +37,10 @@
       end
 
       context 'if Geocoder.search raises an error' do
+        let(:error) {Geocoder::Error.new}
```

Comment:
```
        let(:error) { Geocoder::Error.new 'server error.' }
```

Added some spacing, and an error description that matches how it's used in Geocoder.